pkg/network/netlink/conntrack.go

```diff
@@ -77,7 +97,7 @@ func (c *conntrack) Exists(conn *Con) (bool, error) {
 		return false, err
 	}
 
-	if len(replies) > 0 {
+	if done {
```

Comment:
Is the expectation that a connection that does not exist, will receive `ENOENT`?

Reply:
You know I'm not sure. The previous code seems to suggest that it may not. I can return a count from `ReceiveAndDiscard` for this.

Reply:
Done